### DISCUSSION TITLE
Developer meeting (June 12th): Crazyflie and ROS 2

### DISCUSSION THREAD

**Discussion by knmcguire** [bitcraze/discussions#1379] [Announcements]
The next developer meeting will be on the 12th of June 2024, we will talk about development of ROS 2 and the Crazyflie in light of the Crazyswarm2 project. 

Date: Wednesday, June 12th, 2024
Time: 3 PM Central European Summer Time (check your [timezone](https://www.worldtimebuddy.com/?qm=1&lid=2692969,5368361,1850147,100,14&h=2692969&date=2024-6-11&sln=15-16&hf=0))
Topic: Prototype of forward facing expansion connector
Hosts: @knmcguire
Meeting link: meeting is over, you can find a recording here: https://youtu.be/ODLaRS_n62s
Slides:  [dev meeting 20240612 Crazyflie and ROS2.pdf](https://github.com/user-attachments/files/15820233/dev.meeting.20240612.Crazyflie.and.ROS2.pdf)
Code: https://github.com/knmcguire/crazyflie-ros2-examples

15:00 - 15:30: Presentation and discussion
15:30 - 16:00: Generic support questions

Indicate with 👍 if you are planning to join!

**Comment by knmcguire** [comment]
It's the 12th of June! not the 11th

**Comment by Daneto96** [comment]
Very cool!
I'm looking forward to it :)

**Comment by knmcguire** [comment]
Recording is here: https://youtu.be/ODLaRS_n62s
Code: https://github.com/knmcguire/crazyflie-ros2-examples
